extensions/debuggee/src/org/exist/debuggee/ScriptRunner.java

```diff
@@ -68,8 +68,8 @@ public void stop() {
 	public void run() {
 		try {
 			final Database db = BrokerPool.getInstance();
-			
-            BrokerPool.registerStatusObserver(this);
+
+            db.addStatusObserver(this);
```

Comment:
@shabanovd nothing has really changed around observers... if ***you*** didn't need to unregister **your** observer in the past, then you don't need to now either